Lightning talk for RightsCon2021: deadline Sunday May 9 (#1844)
alan-turing-institute/the-turing-way: conferences/presentations/RIghtsCon-2021-06-07/script.md

So far, contributors to the Guide to Ethical Research have contributed chapters on working with institutional ethics boards, the relationship between ethics and research integrity, and ethics in preclinical research.

But we don't see ethics as a static or one-off part of research.
Ethical research includes every part of research: including making sure your institution is treating its worker, its research participants, and everyone else involved, well. 
The Guide to Ethical Research also includes information on how to advocate for policy change within your institution: if you don't like how something is done, you can change it!
It also covers how to be an activist researcher, and how and why unionisation can be important for data scientists to practice ethically.

The Guide to Ethical Research is a work in progress.
Current chapters-in-progress include how ethical data science fits into legal, policy and human rights frameworks; how to address and overcome challenges in ethical practice; and participatory science methods.

We'd like to invite you to become a contributor to the Turing Way, to help us develop these chapters, and many more. 

> You don't need to have coding experience: just an idea and a willingness to learn. 

Author: ```suggestion
We welcome all kinds of contributions 
Case studies, and lived experiences, are welcome.
You don't need to have coding experience to contribute! 
You just need an idea and a willingness to learn. 
```

We're a supportive community, governed by a code of conduct, and we have a set of contribution guidelines which keeps our community welcoming and friendly.
We have a monthly newsletter that you can sign up for - and a community Slack workspace, that you can sign up for and introduce yourself to. 

> We have a weekly onboarding call, which is open to everyone: come along, say hi, and find out more about the project.

Author: ```suggestion

```

We also run Collaboration Cafes every fortnight to work together on the book.
And twice a year we run Book Dash events, bringing together collaborators from all over the world to work together. 

> We're looking forward to having you join us!

Author: ```suggestion
Or if you'd like to find out more, please come to one of our weekly onboarding calls where you can chat with an experienced community member who will be happy to guide you through the Turing Way
We're looking forward to having you join us!
```

# Script for lightning talk at RightsCon2021

8-10 minute recorded talk, in the Human Rights-Centered Design track - probably about 1500 words

> (possibly also with an AMA during the conference)

Reviewer: Happy to be part of the AMA if you need me and think I can be of help, Laura

## Original pitch
([original application](https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/2021-01-RightsCon.md))
Data scientists make data-driven decisions throughout the lifecycle of their projects. 
It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially, when it impacts people’s lives. 
In this presentation, we will introduce The Turing Way, an open source book project that involves and supports its diverse community in developing and sharing resources that make data science reproducible, ethical, collaborative and inclusive. 
In particular, we will demonstrate the Turing Way guide for Ethical Research, the community behind its development, and its human rights-centred approach to data science ethics as a practice that includes activism for change. 
Most importantly, we will demonstrate how RightsCon attendees can get involved!

(based heavily on [this talk](https://zenodo.org/record/3968454#.YIA1Vmgo-i5) by Malvika Sharan, which was 10 minutes.

## Skeleton:

> 1. Intros and what is the Turing Way? (2 mins)

Reviewer: I think that adding a very short motivation for the talk/the turning way, before jumping into saying what is TTW may help hooking watchers. I will suggest a bit of the script in this direction below

3. What is the Turing Way Guide for Ethical Research? (5 mins)
4. How can you get involved? (3 mins)

## The Turing Way: Building a collaborative guide to ethical data science
### Presenters: Malvika Sharan and Laura Carter

(Introduce speakers)

> 

Reviewer: Data scientists make decisions throughout the lifecycle of their data-based projects.
 
It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially, when it impacts people’s lives.

> The Turing Way is a collaborative book project.

Reviewer: ```suggestion
The Turing Way is a collaborative book project that aims to make reproducible, ethical, and inclusive data science 'too easy not to do.'
```

> It is based at the Alan Turing Institute, the UK's national centre for data science and artificial intelligence. 

Reviewer: ```suggestion
The book is based at the Alan Turing Institute, the UK's national centre for data science and artificial intelligence. 
```

It's an open source, collaborative project: and has contributors from all over the world.

> The aim of the Turing Way is to make reproducible, ethical and inclusive data science 'too easy not to do.'

Reviewer: ```suggestion

```

There are five books in the Turing Way: covering collaboration, project design, reproducibility, communication and outreach, and ethics.

> We're going to focus on the Guide to Ethical Research.

Reviewer: ```suggestion
This talk focuses on the Guide to Ethical Research.
```

The aim of the Guide to Ethical Research is to support data scientists - and researchers using data science - to work in an ethical way.
We start from the position that as data science researchers, we have an obligation to consider the impact of our research. 
So far, contributors to the Guide to Ethical Research have contributed chapters on working with institutional ethics boards, the relationship between ethics and research integrity, and ethics in preclinical research.

But we don't see ethics as a static or one-off part of research.

> Ethical research includes every part of research: including making sure your institution is treating its worker, its research participants, and everyone else involved, well. 

Reviewer 2: Maybe "covers" every part of research, not to repeat "includes"?

The Guide to Ethical Research also includes information on how to advocate for policy change within your institution: if you don't like how something is done, you can change it!

> It also covers how to be an activist researcher, and how and why unionisation can be important for data scientists to practice ethically.

Reviewer: ```suggestion
It also includes information on how to be an activist researcher, and how and why unionisation can be key for an ethical data science practice.
```

The Guide to Ethical Research is a work in progress.
Current chapters-in-progress include how ethical data science fits into legal, policy and human rights frameworks; how to address and overcome challenges in ethical practice; and participatory science methods.

We'd like to invite you to become a contributor to the Turing Way, to help us develop these chapters, and many more. 

> You don't need to have coding experience: just an idea and a willingness to learn. 

Reviewer 3: Could you add in a point about case studies and lived experience being valuable as well?

We're a supportive community, governed by a code of conduct, and we have a set of contribution guidelines which keeps our community welcoming and friendly.
We have a monthly newsletter that you can sign up for - and a community Slack workspace, that you can sign up for and introduce yourself to. 

> We have a weekly onboarding call, which is open to everyone: come along, say hi, and find out more about the project.

Reviewer: I would highlight the onboarding call as the most accessible point to enter the community because there is an experienced person in the other end that can guide you and help you navigate the "information sea" that TTW has become :-)

I would move it last on the list on ways to join and say something along the lines of "If none of the above options ring a bell or prompt you to join TTW, please come to one of our weekly onboarding calls where you can chat with an experienced community member who will be happy to guide you through TTW."

# Script for lightning talk at RightsCon2021

8-10 minute recorded talk, in the Human Rights-Centered Design track - probably about 1500 words
(possibly also with an AMA during the conference)

## Original pitch
([original application](https://github.com/alan-turing-institute/the-turing-way/blob/master/conferences/abstracts/2021-01-RightsCon.md))
Data scientists make data-driven decisions throughout the lifecycle of their projects. 
It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially, when it impacts people’s lives. 
In this presentation, we will introduce The Turing Way, an open source book project that involves and supports its diverse community in developing and sharing resources that make data science reproducible, ethical, collaborative and inclusive. 
In particular, we will demonstrate the Turing Way guide for Ethical Research, the community behind its development, and its human rights-centred approach to data science ethics as a practice that includes activism for change. 

> Most importantly, we will demonstrate how RightsCon attendees can get involved!

Author: ```suggestion
Most importantly, we will tell you how you can get involved!
```

(based heavily on [this talk](https://zenodo.org/record/3968454#.YIA1Vmgo-i5) by Malvika Sharan, which was 10 minutes.

## Skeleton:
1. Intros and what is the Turing Way? (2 mins)
3. What is the Turing Way Guide for Ethical Research? (5 mins)
4. How can you get involved? (3 mins)

## The Turing Way: Building a collaborative guide to ethical data science
### Presenters: Malvika Sharan and Laura Carter

(Introduce speakers)

> Data scientists make decisions throughout the lifecycle of their data-based projects.

Author: ```suggestion
As data scientists, we make decisions throughout the lifecycle of our data-based projects.
```

> It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially when it impacts people’s lives.

Author: ```suggestion
As scientists, we have a duty to ensure that our work meets the highest scientific standards.
We also have an obligation to ensure that our work maintains the highest level of moral integrity: especially when it impacts people’s lives.
This impact on lives can be through the data we collect and use, or through the data products we build.
It can also be through the policies that we follow at our universities or institutions, and the working conditions that we create in our labs.

```

> It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially, when it impacts people’s lives.

Author: ```suggestion
It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially when it impacts people’s lives.
```

> It is their duty to ensure that their work maintains a high level of moral integrity as well as the highest scientific standards, especially, when it impacts people’s lives.

Author: ```suggestion
As data scientists, we have an duty to ensure that our work meets the highest scientific standards. 
We also have an obligation that our work maintains a high level of moral integrity: especially when it impacts - directly or indirectly - people's lives.
```

The Turing Way is a collaborative book project that aims to make reproducible, ethical, and inclusive data science 'too easy not to do.'
The book is based at the Alan Turing Institute, the UK's national centre for data science and artificial intelligence. 
It's an open source, collaborative project: and has contributors from all over the world.

> 

Author: ```suggestion
```

This talk focuses on the Guide to Ethical Research.

The aim of the Guide to Ethical Research is to support data scientists - and researchers using data science - to work in an ethical way.
We start from the position that as data science researchers, we have an obligation to consider the impact of our research. 
So far, contributors to the Guide to Ethical Research have contributed chapters on working with institutional ethics boards, the relationship between ethics and research integrity, and ethics in preclinical research.

But we don't see ethics as a static or one-off part of research.
Ethical research covers every part of research: including making sure your institution is treating its worker, its research participants, and everyone else involved, well. 
The Guide to Ethical Research also includes information on how to advocate for policy change within your institution: if you don't like how something is done, you can change it!
It also includes information on how to be an activist researcher, and how and why unionisation can be key for an ethical data science practice.

The Guide to Ethical Research is a work in progress.

> Current chapters-in-progress include how ethical data science fits into legal, policy and human rights frameworks; how to address and overcome challenges in ethical practice; and participatory science methods.

Author: ```suggestion
Current chapters-in-progress include how ethical data science fits into legal, policy and human rights frameworks; how to address and overcome challenges in ethical practice; and participatory science methods.
We also include case studies, like the successful campaign against Robodebt in Australia.
And we have written about the history of unionisation by scientific workers - and how you can join and support a union yourself!
```
